test/functional/Jsr292/playlist.xml

```diff
@@ -100,6 +100,13 @@
 		<subsets>
 			<subset>11+</subset>
 		</subsets>
+		<!-- 
+			TODO: Further investigation is needed running against non-openj9 sdk
```

Comment:
Please update the message to:
TODO: Further investigation is needed `for` running against non-openj9 sdk